Formula/dbus.rb

```diff
@@ -35,6 +36,12 @@ class Dbus < Formula
     sha256 "a8aa6fe3f2d8f873ad3f683013491f5362d551bf5d4c3b469f1efbc5459a20dc"
   end
 
+  # add macports patch to disable external authentication
+  patch :p0 do
+    url "https://raw.githubusercontent.com/macports/macports-ports/master/devel/dbus/files/patch-configure.diff"
```

Comment:
Using `master` here seems like asking for trouble.

Reply:
Fixed